editor/debugger/script_editor_debugger.h

```diff
@@ -150,6 +150,8 @@ class ScriptEditorDebugger : public MarginContainer {
 	bool can_debug = false;
 
 	bool live_debug;
+	bool debug_step_or_next;
+	bool from_debug_enter;
```

Comment:
Not sure about this two names. They are not very helpful when understanding what they are doing.

Reply:
Also, you forgot to initialize them either in the `ScriptEditorDebugger` constructor or here.

Reply:
they correspond to messages and function names, corrected